src/rules/checkLinesAlignment.js

```diff
@@ -0,0 +1,210 @@
+import {
+  set,
+} from 'lodash';
+import iterateJsdoc from '../iterateJsdoc';
+
+/**
+ * Aux method until we consider the dev envs support `String.prototype.matchAll` (Node 12+).
+ *
+ * @param {string}   string   String that will be checked.
+ * @param {RegExp}   regexp   Regular expression to run.
+ * @param {Function} callback Function to be called each iteration.
+ * @param {int}      limit    Limit of matches that we want to exec.
+ *
+ * TODO [engine:node@>=12]: Remove function and use `String.prototype.matchAll` instead.
```

Comment:
My apologies--I was mistaken about `@todo` not being supported. So please do change `TODO` to `@todo`.

Reply:
No worries! Thank you for check that! =)
https://github.com/gajus/eslint-plugin-jsdoc/pull/636/commits/af433fa79ec5197a76e5707784e528f57b91b8be